docs/docs/configuration/nextjs.md

```diff
@@ -1,5 +1,60 @@
 # Next.js
 
+## `getServerSession`
+
+> 🚧 This is experimental and may be removed in the future.
+
+When calling from server side i.e. in API routes or in `getServerSideProps`, we recommend use this function instead of `getSession` to retrieve the `session` object. This method is especially useful when you are using NextAuth.js with a database. Otherwise, if you only want to get the token, see [`getToken`](tutorials/securing-pages-and-api-routes#using-gettoken).
```

Comment:
```suggestion
When calling from server-side i.e. in API routes or in `getServerSideProps`, we recommend using this function instead of `getSession` to retrieve the `session` object. This method is especially useful when you are using NextAuth.js with a database. Otherwise, if you only want to get the token, see [`getToken`](tutorials/securing-pages-and-api-routes#using-gettoken).
```